testcases/multizone_office_simple_hydronic/models/library_versions.json

```diff
@@ -0,0 +1,4 @@
+{
+"IDEAS_COMMIT": "cd7ce8c44d597c6ca02a0db4c90cbf2bed29754d",
+"BUILDINGS_COMMIT": "ea147aeee6e0a0239f1b48f5cd75f7e72963bf73"
+}
```

Comment:
@icupeiro Are these commits corresponding to specific library version tags (e.g. Buildings v9.0.0) or are they master branch commits between versions?

Reply:
These are the latest master branches (from yesterday). In fact, IDEAS was recently updated so that some material properties were modified using better sources. See https://github.com/open-ideas/IDEAS/commit/cd7ce8c44d597c6ca02a0db4c90cbf2bed29754d. 

Needless to say, documentation has been properly updated.